### DISCUSSION TITLE
Site oEmbed image could do with update?

### DISCUSSION THREAD

**Discussion by rodneytamblyn** [datalust/seq-tickets#2145] [Ideas]
https://datalust.co/seq

![image](https://github.com/datalust/seq-tickets/assets/1562956/7e2a58a8-d997-48a7-8126-4b1fd4a3d9c5)

**Comment by liammclennan** [comment]
Hi Rodney,

I'm not sure what this is about. Can you please add some more detail?

**Comment by rodneytamblyn** [comment]
If you link a URL pointing at SEQ website, that's the image it shows.  Try it yourself, paste the URL above into a slack channel for example.  To update your oembed endpoint content, see: https://oembed.com/

For a walkthrough in adding provider info to your website see: https://timnash.co.uk/becoming-oembed-provider/

**Reply by liammclennan** [reply]
Hi Rodney,

Thanks for letting us know.
